instances/unitOfMeasurement/embryonicDay.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/embryonicDay",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "name": "embryonic day",
+  "definition": "'Embryonic day' is a specific unit to measure the time pasted since fertilization of an egg cell.",
```

Comment:
I would be even more specific:
"'Embryonic day' is a specific unit to measure the developmental stage of an embryo, starting with fertilization (1st embryonic day) and ends with birth."

@UlrikeS91 What do you think?

Reply:
Not sure if "measure the developmental stage of an embryo" is 100% correct. The embryonic days of course coincide with developmental stages but are different for different species (since pregnany durations and embryo development differ). However, for one species these embryonic days are used exactly to refer to the developmental stage without having to describe it or use many words. 
You're definition is definitely better than mine. So, I changed it. We can ask e.g. Ingvild later on if she has an even better suggestion :)